src/components/HomepageFeatures/index.tsx

```diff
@@ -63,13 +63,13 @@ const FeatureList: FeatureItem[] = [
     ),
   },
   {
-    title: 'Use WebAssembly(WASM)',
+    title: 'Use Swanky to develop WebAssembly smart contracts',
```

Comment:
Or:
Use WASM with Swanky

Your current title is a little to long.

Reply:
If we change this name, we also have to change the title name in the front page and side bar.
I can do this also if it sounds ok for you. @andabakus

Reply:
@fiexer - changed. thanks!

@SUGURUSEO - Feel free to work on it. Regarding WASM section, It need a lot of work in itself I think (Swanky is now somewhere on third level of docs, should be reorganized chapter)